whitepaper/chapter01.md

```diff
@@ -0,0 +1,150 @@
+
+<a name="1.2"></a>
+## 1.2 What is Cloud Native?
+
+After a collaborative discussion, the CNCF published the following [definition](../../../../toc/blob/master/DEFINITION.md) of cloud native:
+
+“Cloud native technologies empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach. These techniques enable loosely coupled systems that are resilient, manageable, and observable. Combined with robust automation, they allow engineers to make high-impact changes frequently and predictably with minimal toil.”
+
+<a name="1.3"></a>
+## 1.3 Defining Cloud Native Systems
+
+There are some additional key principles to cloud native that are important to grasp in order to think more broadly about architecting cloud native technologies for complex systems.A fundamental principle of cloud native for telco apps is that their underlying telco systems conform to the core precepts of cloud native architecture. The lack of exceptions makes the rule, because the entire advantage of cloud native is driven by decomposability and abstraction from physical infrastructure. By deviating even slightly from these core principles, the service provider stands to lose the workload portability and minimal toil that drew them to cloud native approaches in the first place.
+
+<a name="1.3.1"></a>
+### 1.3.1 What "loosely coupled" means in cloud native systems
+
+Cloud native systems work on the principle of breaking a larger monolithic application in to a fine tuned components  that is based on have a clear separation between their microservices. The separation is based on a technology like containers which will allow to deploy each of this split software component on an independent set of container infrastructure. Cloud native applications and systems generally benefit from segregating as many services as possible into different containers. Additional benefits of cloud native architectures include scalability (vertical and horizontal) per microservice. This simplifies deployment in a wide variety of infrastructure environments, ranging from endpoint device to edge station to large data centres. It also allows service providers to deploy resources in the most efficient manner possible while maintaining necessary service levels for regulated and unregulated service types.
+
+The delivery of microservice software harnesses the existing group boundaries within an organisation and works with, not against, the different rates of change and business capabilities residing within those boundaries. This allows events such as security patches to be rolled out faster and easier because they don't have to be deployed in lock-step with other parts of the application.
```

Comment:
This entire next set of paragraphs has duplication and overlap.  It needs to be reworked  and collapsed